### ISSUE TITLE
Use flavor name instead of build type for ribbon text

### ISSUE THREAD

**Issue by gpeal** [akaita/easylauncher-gradle-plugin#12]
In our project, the build type doesn't really matter because the only debuggable builds are of the debug build type. However, we have a number of flavors used just for development and that's where the real value of easylauncher is for us. Can there be an option to show the flavor instead of build type?

**Comment by akaita** [comment]
True, that is a very sensible request.

Choosing the text to add into the ribbon could get tricky.  Let me borrow your brains for this...   

I'm thinking that this could mean that all `qa` flavors get a ribbon which says "qa":
```
easylauncher {
    productFlavors {
        qa {
            filter = redRibbonFilter()
        }
    }
}
```

And this could mean that all `beta` types get a ribbon which says "beta":
```
easylauncher {
    buildTypes {
        beta {
            filter = redRibbonFilter()
        }
    }
}
```

I think that'd make sense.

btw: I plan on allowing custom text into the ribbon #1 (this one is easy, so it will probably come in a couple of days). That should cover all use cases right away.

**Comment by akaita** [comment]
Oh, my. I've been awake for too long 😅 

You can already use this to give a red ribbon to all `qa` builds,  so you can get the results you are looking for:
```groovy
easylauncher {
    productFlavors {
        qa {
            filter = redRibbonFilter()
        }
    }
}
```

I now understand you were thinking about the default behaviour of the plugin.  

In this case, I think sticking to buildTypes makes more sense in most people's case.  
I can see that an option to use flavor names instead of buildType names could be useful, though:

```groovy
easylauncher {
    flavorNaming = true   // default: false
}
```

I'll leave the issue open to get it done at some point.

**Comment by gpeal** [comment]
Thanks! This configuration will make this plugin extremely useful to us :)

**Comment by akaita** [comment]
I decide that it'd be clearer to rename the option to:

```groovy
easylauncher {
    defaultFlavorNaming = true   // default: false
}
```

Released a few minutes ago in v1.1.0 🎉
